telegram/ext/_chatjoinrequesthandler.py

```diff
@@ -43,6 +54,15 @@ async def callback(update: Update, context: CallbackContext)
 
             The return value of the callback is usually ignored except for the special case of
             :class:`telegram.ext.ConversationHandler`.
+        chat_id (:obj:`int` | Collection[:obj:`int`], optional): Filters requests to allow only
+            those which are asking to join the specified chat ID(s).
+
+            .. versionadded:: 20.0
+        username (:obj:`str` | Collection[:obj:`str`], optional): Filters requests to allow only
+            those which are asking to join the specified username(s).
+            Leading ``'@'`` in username will be discarded.
```

Comment:
We dont mention this in filters, and I dont think its necessary. Doesn't matter what the user input, the code just handles it. So I would remove this comment.

Reply:
Remove `Leading ``'@'`` in username will be discarded.` from comment?